Userland/Libraries/LibWebView/WebContentClient.cpp

```diff
@@ -395,6 +395,16 @@ void WebContentClient::did_request_file(DeprecatedString const& path, i32 reques
 {
     if (m_view && m_view->on_request_file)
         m_view->on_request_file(path, request_id);
+
+    // FIXME: Hax to make local workers work, remove this!
+    if (!m_view) {
+        auto file = Core::File::open(path, Core::File::OpenMode::Read);
+
+        if (file.is_error())
+            async_handle_file_return(file.error().code(), {}, request_id);
+        else
+            async_handle_file_return(0, IPC::File(*file.value()), request_id);
+    }
```

Comment:
This is particularly egregious, but the alternative is to invent that "WorkerView" thing that feels like overkill.